roboflow/core/version.py

```diff
@@ -92,10 +92,16 @@ def download(self, model_format=None, location=None):
         """
         Download and extract a ZIP of a version's dataset in a given format
 
-        :param model_format: A format to use for downloading
-        :param location: An optional path for saving the file
+        Args:
+            model_format: A format to use for downloading
+            location: An optional path for saving the file
 
-        :return: Dataset
+        Returns:
+            Dataset
```

Comment:
This will be never parsed, need to specify where it comes from (the module path)